package.json

```diff
@@ -34,6 +34,7 @@
     "css-loader": "^0.26.2",
     "extract-text-webpack-plugin": "^3.0.0",
     "file-loader": "^0.10.1",
+    "fork-ts-checker-webpack-plugin": "^0.2.7",
```

Comment:
Will get a linting error then. I even reported that in Realytics/fork-ts-checker-webpack-plugin#34 but it seems to be right like that. 

Honestly I can not argue much about it since I still don't get right eslint rules :(